launch/tier4_autoware_api_launch/launch/autoware_api.launch.xml

```diff
@@ -1,45 +1,32 @@
 <launch>
-  <arg name="respawn_rosbridge" default="true"/>
-  <arg name="launch_path_distance_calculator" default="false"/>
+  <!-- AD API options -->
   <arg name="launch_default_ad_api" default="true"/>
   <arg name="launch_rviz_adaptors" default="true"/>
-  <!-- external api adaptor -->
-  <arg name="launch_calibration_status_api" default="false"/>
-  <arg name="launch_start_api" default="true"/>
+
+  <!-- rosbridge options -->
+  <arg name="rosbridge_enabled" default="false"/>
+  <arg name="rosbridge_respawn" default="true"/>
+  <arg name="rosbridge_max_message_size" default="10000000"/>
```

Comment:
This default size was 1000000000 in autoware_launch.
Is there any reason to reduce this?

Reply:
Before the change, this parameter is not set in autoware.universe, so it defaults to 10000000 ([this value](https://github.com/RobotWebTools/rosbridge_suite/blob/ros2/rosbridge_server/launch/rosbridge_websocket_launch.xml#L12)). 1000000000 is a TIER IV repository setting.